### PR TITLE
Add wsearch directive doc

### PR DESCRIPTION
**Pull request by yipenghe**
#1830 
Also changed documentation => CLI documentation and linked to the CLI reference page

Vscode Preview:
Screenshot from 2020-01-19 16-21-19

### CODE REVIEW

File: docs/Worksheet-Markdown.md [hunk 1]
```diff
@@ -210,7 +210,15 @@ To reference a set of bundles by a search criteria:
 
     % search <keyword-1> ... <keyword-n>
 
-See documentation for `cl search` for more information.
+See [CLI documentation](https://codalab-worksheets.readthedocs.io/en/latest/CLI-Reference/) for `cl search` for more information.
```
Comment (percyliang) [inline docs/Worksheet-Markdown.md]: "for more information about `cl search`"

File: docs/Worksheet-Markdown.md [hunk 2]
```diff
@@ -210,7 +210,15 @@ To reference a set of bundles by a search criteria:
...
+
+### Displaying a dynamic set of worksheets
+
+To reference a set of worksheets by a search criterira:
+    
+    % wsearch <keyword-1> ... <keyword-n>
+
+See [CLI documentation](https://codalab-worksheets.readthedocs.io/en/latest/CLI-Reference/) for `cl wsearch` for more information.
```
Comment (percyliang) [inline docs/Worksheet-Markdown.md]: same comment as above

File: docs/Worksheet-Markdown.md [hunk 3]
```diff
@@ -210,7 +210,15 @@ To reference a set of bundles by a search criteria:
 
     % search <keyword-1> ... <keyword-n>
 
-See documentation for `cl search` for more information.
+See [CLI documentation](https://codalab-worksheets.readthedocs.io/en/latest/CLI-Reference/) for `cl search` for more information.
+
+### Displaying a dynamic set of worksheets
+
+To reference a set of worksheets by a search criterira:
```
Comment (percyliang) [inline docs/Worksheet-Markdown.md]: typo

**Review by percyliang — APPROVED** [review]

File: docs/Worksheet-Markdown.md (hunk 1 above)
Comment (yipenghe) [reply docs/Worksheet-Markdown.md]: Maybe should be 'See CLI documentation about 'cl search' for more information'?

Comment (percyliang) [reply docs/Worksheet-Markdown.md]: Sure

**Pull request merged**